collectors/apps.plugin/apps_plugin.c

```diff
@@ -4183,7 +4198,7 @@ int main(int argc, char **argv) {
             printf("DISABLE\n");
             exit(1);
         }
-
```

Comment:
Why you deleted this newline?

Reply:
For me the line is showing fine, maybe you saw an oudated commit? (sorry I commeted quickly)